.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,60 @@
+name: Code Coverage Report
+
+on: [push, pull_request]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
```

Comment:
Configuring the project with
```
cmake -B cmake-build-unit-tests -S executables/unitTest -DBUILD_UNIT_TESTS=ON
```
Without setting build type to "Debug" will give the following output
```
CMake Warning at openbsw/admin/cmake/CodeCoverage.cmake:214 (message):
  Code coverage results with an optimised (non-Debug) build may be misleading
Call Stack (most recent call first):
~/openbsw/CMakeLists.txt:35 (include)
  CMakeLists.txt:21 (include)
  ```
  Implying that the project should be configured for debug in order to generate code coverage data.

Reply:
yes...please set the build type to `Debug`

Reply:
Yes thank you for letting me know